Author: gregor herrmann <gregoa@debian.org>
Description: bump required perl version, otherwise Test::MinimumVersion whines and tests fail
Bug: #521580

--- a/META.yml
+++ b/META.yml
@@ -21,5 +21,5 @@
   Class::Adapter: 1.00
   Module::Pluggable: 2.5
   Params::Util: 0.14
-  perl: 5.005
+  perl: 5.00503
 version: 0.05
